doc/tor.md

```diff
@@ -11,12 +11,17 @@ configure Tor.
 The first step is running Bitcoin Core behind a Tor proxy. This will already anonymize all
 outgoing connections, but more is possible.
 
-	-proxy=ip:port  Set the proxy server. If SOCKS5 is selected (default), this proxy
+	-proxy=ip:port  Set a proxy server. If SOCKS5 is selected (default), this proxy
 	                server will be used to try to reach .onion addresses as well.
+	                You have to use -noonion or -onion=0 to explicitly disable
+	                outbound access to onion services.
 
 	-onion=ip:port  Set the proxy server to use for Tor onion services. You do not
-	                need to set this if it's the same as -proxy. You can use -noonion
+	                need to set this if it's the same as -proxy. You can use -onion=0
 	                to explicitly disable access to onion services.
+	                Note: Only the -proxy option will set the proxy used for DNS requests.
+	                With this option these will not route over Tor.
+	                So please use -proxy if you have privacy concerns with this.
```

Comment:
This seems out of place.  If no proxy is set then *all* traffic, except traffic going to hidden services, is in the clear. This edit makes it sound otherwise.

Reply:
Its more about Tor options doc than anything else, please hint wording alternatives..

Please note we try to highlight here that -onion in contrast to -proxy sets no name proxy and also up till now the default Tor proxy that would be created in torcontrol is also a leaking ip over DNS ip4/6 calls and will probably leak when seeding at first start.

The internal `nameproxy` to try to route all allowed traffic over Tor is only used if -proxy is defined.

Reply:
I'm having a hard time following you.   Onion related requests are not leaked when you set -onion.